addons/fortify/functions/fnc_deployObject.sqf

```diff
@@ -50,17 +50,13 @@ private _mouseClickID = [_player, "DefaultAction", {GVAR(isPlacing) == PLACE_WAI
     params ["_args", "_pfID"];
     _args params ["_unit", "_object", "_cost", "_mouseClickID"];
 
-    if ((_unit != ACE_player) || {isNull _object} || {!([_unit, _object, []] call ACEFUNC(common,canInteractWith))} || {!([_unit, _cost] call FUNC(canFortify))}) then {
+    if (_unit != ACE_player || {isNull _object} || {!([_unit, _object, []] call ACEFUNC(common,canInteractWith))} || {!([_unit, _cost] call FUNC(canFortify))}) then {
         GVAR(isPlacing) = PLACE_CANCEL;
     };
 
-    if (GVAR(isPlacing) == PLACE_APPROVE) then {
-        // Run custom deploy handlers
-        private _someReturnFalse = {if ([_unit, _object, _cost] call _x isEqualTo false) exitWith {1}} count GVAR(deployHandlers);
-        if (_someReturnFalse isEqualTo 1) then {
-            // do not allow placement
-            GVAR(isPlacing) = PLACE_WAITING;
-        };
+    // If place approved, verify deploy handlers
+    if (GVAR(isPlacing) == PLACE_APPROVE && {GVAR(deployHandlers) findIf {!([_unit, _object, _cost] call _x)} > -1}) then {
```

Comment:
Slightly more robust way might be to use `isEqualTo`, but it's probably not needed
```
x0 = [{true}, {nil}, {5}, {false}]; 
x1 = x0 findIf {!(call _x)};             // nil
x2 = x0 findIf {false isEqualTo call _x} // 3
```